### PR TITLE
feat：Support SkyWalking to collect metrics

### PR DESCRIPTION
**Pull request by cobb-tx**
### Ⅰ. Describe what this PR did
helm 模板支持envoy metrics 推送给skywalking

### Ⅱ. Does this pull request fix one issue?

https://github.com/alibaba/higress/issues/56

### Ⅳ. Describe how to verify it
本地验证的

### CODE REVIEW

File: helm/higress/templates/configmap.yaml [hunk 1]
```diff
@@ -115,4 +115,16 @@ data:
 {{- include "mesh" . }}
 {{- end }}
 ---
-
+  {{- if .Values.gateway.customBootstrap.enabled  }}
```
Comment (johnlanni) [inline helm/higress/templates/configmap.yaml:118]: It is recommended to use `enableSkywalking` to control this logic, which is easier to understand.

File: helm/higress/templates/configmap.yaml [hunk 2]
```diff
@@ -115,4 +115,16 @@ data:
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: higress-custom-bootstrap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gateway.labels" . | nindent 4 }}
+data:
+  custom_bootstrap.json:
+    "{\"stats_sinks\":[{\"name\":\"envoy.metrics_service\",\"typed_config\":{\"@type\":\"type.googleapis.com/envoy.config.metrics.v3.MetricsServiceConfig\",\"transport_api_version\":\"V3\",\"grpc_service\":{\"envoy_grpc\":{\"cluster_name\":\"service_skywalking\"}}}}],\"static_resources\":{\"clusters\":[{\"name\":\"service_skywalking\",\"type\":\"LOGICAL_DNS\",\"connect_timeout\":\"5s\",\"http2_protocol_options\":{},\"dns_lookup_family\":\"V4_ONLY\",\"lb_policy\":\"ROUND_ROBIN\",\"load_assignment\":{\"cluster_name\":\"service_skywalking\",\"endpoints\":[{\"lb_endpoints\":[{\"endpoint\":{\"address\":{\"socket_address\":{\"address\":\"{{ .Values.gateway.customBootstrap.skywalking.address }}\",\"port_value\":\"{{ .Values.gateway.customBootstrap.skywalking.port }}\"}}}}]}]}}]}}"
```
Comment (johnlanni) [inline helm/higress/templates/configmap.yaml:128]: This configuration is difficult to maintain, please use helm functions such as `toJson` to optimize it.

**Review by johnlanni — CHANGES_REQUESTED** [review]

File: helm/higress/values.yaml [hunk 3]
```diff
@@ -318,6 +318,13 @@ global:
   # The default value is "" and when caName="", the CA will be configured by other
   # mechanisms (e.g., environmental variable CA_PROVIDER).
   caName: ""
+
+  customBootstrap:
+    enabled: false
+    skywalking:
+      address: ""
+      port: ""
```
Comment (johnlanni) [inline helm/higress/values.yaml:326]: This configuration should be independent of bootstrap.

File: helm/higress/templates/configmap.yaml (hunk 2 above)
Comment (cobb-tx) [reply helm/higress/templates/configmap.yaml]: ok

**Comment by Xunzhuo** [comment]
plz sign your commit and rebase the conflicts first @cobb-tx

**Pull request closed without merging**